lib/tacos/dependent_slices.py

```diff
@@ -243,11 +243,16 @@ def lines_to_paths(lines: Iterable[str]) -> Generator[OSPath]:
 
 def main() -> int:
     import fileinput
+    from sys import argv
+
+    args = argv[1:]
+    # need to modify argv for fileinput to work
+    del argv[1:]
 
     fs = FileSystem.from_git()
     modified_paths = lines_to_paths(fileinput.input(encoding="utf-8"))
```

Comment:
fileinput is only useful where we want to not care whether the user is using stdin or positional arguments.
Since we can no longer use positional arguments for file inputs, `fileinput` is unecessary complextiy and this would be expressed more simply as:

```suggestion
    from sys import stdin
    ...
    modified_paths = lines_to_paths(stdin)
```

and then there's no need to modify argv, either.